src/classic/clvm_tools/stages/stage_2/optimize.rs

```diff
@@ -112,7 +156,7 @@ fn constant_optimizer<'a>(
             );
             let r1 = res.1;
             let _ = if DIAG_OPTIMIZATIONS {
-                print!(
+                println!(
                     "CONSTANT_OPTIMIZER {} TO {}\n",
```

Comment:
did you mean to leave the explicit new-line in the format string as you changed it from `print!()` to `println!()`?

Reply:
these are just diagnostics i honestly didn't care
this was a clippy warning at one point